site/load-chart.js

```diff
@@ -125,6 +142,10 @@ function setupChart () {
             const score_2013 = s[area_index + 3]
             const score_2020 = s[stride + area_index + 5]
             const date = parseDateString(s[0])
+            const monthsToSubtract = dateRanges[chosen_date]
+            if (monthsToSubtract && date < new Date(maxDate.getFullYear(), maxDate.getMonth() - monthsToSubtract, 1)) {
```

Comment:
Might be better to move the initialization of the Date outside the loop e.g
```
let minDate = null
// check monthsToSubtract and initialize minDate if non-null
```

and inside the loop we can just do
```
if (date < minDate) {
  continue
}
```